ddpui/tests/common/fixtures/chart_ui_payloads.py

```
@@ -0,0 +1,200 @@
+"""Real UI payloads webapp_v2 sends for chart create — one per chart_type.
+
+Captured from live smoke-test charts (55–60). When the UI changes shape, update
```

Ruff (0.15.15)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)